src/cfn.py

```diff
@@ -266,7 +266,7 @@ def ssh(stackname, node=None, username=DEPLOY_USER):
     if not instances:
         return
     public_ip = _pick_node(instances, node).public_ip_address
-    _interactive_ssh("ssh %s@%s -i %s" % (username, public_ip, USER_PRIVATE_KEY))
+    _interactive_ssh("ssh -o StrictHostKeyChecking=no %s@%s -i %s" % (username, public_ip, USER_PRIVATE_KEY))
```

Comment:
:thinking: if the goal is to improve CI, this task shouldn't be in use there as it is for interactive SSH sessions rather than executing commands over SSH like Jenkins does

Reply:
the intent of the ticket was to provide two fixes, however after going through the lifecycle code one of the checks was already being done. 

The problem I've encountered twice now in Alfred would be fixed by the above if Alfred were using ssh. Thinking about for a second, Alfred is probably using `cfn.cmd`, right?

I'd leave the above change in, it's slightly convenient

Reply:
> Alfred is probably using cfn.cmd

Mostly `cfn.switch_revision_update_instance`